lib/src/install.rs

```diff
@@ -102,6 +102,18 @@ pub(crate) struct InstallTargetOpts {
     pub(crate) skip_fetch_check: bool,
 }
 
+#[derive(clap::Args, Debug, Clone, Serialize, Deserialize)]
+pub(crate) struct InstallSourceOpts {
+    /// Install the system from an explicitly given source.
+    ///
+    /// By default, bootc install and install-to-filesystem assumes that it runs in a podman container, and
+    /// it takes the container image to install from the podman's container registry.
+    /// If --source-imgref is given, bootc uses it as the installation source, instead of the behaviour explained
+    /// in the previous paragraph. See skopeo(1) for accepted formats.
+    #[clap(long, hide = true)]
```

Comment:
But documenting seems to clash with `hide = true`.  Either it's something we support ~forever or it's a hidden thing that could be removed in theory.  I guess given the state, we should probably lift `hide = true` right?

Reply:
I removed `hide = true`.